scalafix-core/shared/src/main/scala/scalafix/internal/config/NoInferConfig.scala

```diff
@@ -0,0 +1,22 @@
+package scalafix.internal.config
+
+import metaconfig._
+import org.langmeta._
+
+case class NoInferConfig(
```

Comment:
God there is an incredible amount of ceremony involved around adding configuration, we should try to come up with something easier 🤔